blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -169,6 +169,9 @@ Changes
 Fixes
 =====
 
+- Fixed a ``ClassCastException`` occurring when serialising the result for
+  ``pgtype.typlen`` using the Postgres Wire Protocol.
```

Comment:
This should be a bit more user-facing, e.g.:
Fixed a ``ClassCastException`` which could occur when using prepared statements over the Postgres Wire Protocol.

Reply:
ok, thanks for the comment. 
So, shouldn't I provide specific info eg mention pgtype.typlen?

Reply:
Ah right, I've read the fix wrong, the bug is not on describeMessage.
So maybe:
Fixed a ``ClassCastException`` which could occur when selecting ``pg_catalog.pg_type.typlen``.